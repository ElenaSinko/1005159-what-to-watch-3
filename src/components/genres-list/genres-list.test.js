import React from "react";
import renderer from "react-test-renderer";
import GenresList from "./genres-list.jsx";

const genres = [`a`, `b`, `c`];

it(`Render App`, () => {
  const tree = renderer
    .create(<GenresList
      onGenreTitleClick={() => {}}
      genres={genres}
    />)
    .toJSON();

  expect(tree).toMatchSnapshot();
});

