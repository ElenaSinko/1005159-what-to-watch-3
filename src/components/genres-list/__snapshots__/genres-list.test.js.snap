// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render App 1`] = `
<ul
  className="catalog__genres-list"
>
  <li
    className="catalog__genres-item"
    onClick={[Function]}
  >
    <a
      className="catalog__genres-link"
    >
      a
    </a>
  </li>
  <li
    className="catalog__genres-item"
    onClick={[Function]}
  >
    <a
      className="catalog__genres-link"
    >
      b
    </a>
  </li>
  <li
    className="catalog__genres-item"
    onClick={[Function]}
  >
    <a
      className="catalog__genres-link"
    >
      c
    </a>
  </li>
</ul>
`;
